### DISCUSSION TITLE
Arc-like UI

### DISCUSSION THREAD

**Discussion by Axenide** [zen-browser/desktop#1855] [Ideas]
Hi, I absolutely love Zen Browser. It has been the best browsing experience I had in a long time.
I've been customizing my UI and found that I can add a search bar to the tab sidebar, which is great. Though I tried to do the same with the URL bar with no luck.
I haven't really used Arc for more than a minute, but I liked this UI, where you can have basically everything in this sidebar.
I would like to be able to recreate this, so everything stays in this vertical layout, saving even more vertical space.
![](https://github.com/user-attachments/assets/54eb98b4-ea5d-455d-960f-8594ee608a72)
Though I understand that Zen isn't trying to be an Arc clone, and I think that's great.

**Comment by indexxing** [comment]
I personality like the UI switch from Arc -> Zen, but I'm sure some people would appreciate an Arc-like design. I've personally not tried this theme but the image seems to be Arc-like: https://zen-browser.app/themes/bed8c922-616a-4165-8c86-6822ccf478ad

**Reply by Axenide** [reply]
Yeah, that theme mostly makes color changes, but it doesn't change the UI layout. However I appreciate the recommendation. :)

**Reply by indexxing** [reply]
Oh sorry, I didn't install the theme I just looked at the picture. Hope you find what you're looking for though!

**Comment by ThatAmuzak** [comment]
There's one thing you could consider. Installing and enabling the following two zen mods:

- [Super URL Bar](https://zen-browser.app/themes/d93e67f8-e5e1-401e-9b82-f9d5bab231e6) (Set blurred background to enabled and remove the border)
- [Floating URLbar](https://zen-browser.app/themes/83a641f7-eca9-4c0f-91af-45627bef0539)

Creates a Arc like search bar in the center with the blurred background, and can be triggered with the shortcut control (or command) + L. With this you can keep the search bar to the left, and bring up the url bar itself with a shortcut at the center of the screen. It's more of a workaround than a solution, but it might give you some ideas :D

**Reply by Axenide** [reply]
Thanks, I am actually using them lol, but I wanted to be able to move the URL bar to the tab bar heh.

**Comment by jeanlucthumm** [comment]
For everything but the URL, I think mine's pretty close:

![image](https://github.com/user-attachments/assets/5d40e10b-851e-49ac-a557-7a795d37e6a4)

Use these two zen mods:

![image](https://github.com/user-attachments/assets/3fa85086-3bcc-4109-9865-44b110fcca65)

Set the tabs to always expand:

![image](https://github.com/user-attachments/assets/6c8bd0fc-d7cf-4651-9a1b-450203126741)

The "favorites" you get by pinning tabs

**Reply by Axenide** [reply]
Nice, also you don't need SuperGradient anymore since now it is native!

**Reply by emilekberg** [reply]
What's lacking here is the two way of pinning tabs. 
One where they are more or less normal "tabs" but sticky, that does not automatically close after x amount of time, and then obviously the automatically closing the tabs.

**Reply by jeanlucthumm** [reply]
@Axenide I looked through the settings and can't seem to find the native version of that, would you mind pointing me to the right place?

**Reply by Axenide** [reply]
@jeanlucthumm Right-click on the tab sidebar and go to "Change theme colors" (I have it in Spanish so it might say something else).

If you don't have the option, update.

**Comment by eden881** [comment]
Maybe it wasn't like this when you posted, but I believe Zen is already doing what you ask:

![1](https://github.com/user-attachments/assets/a1e883bd-4f2b-421e-9b05-7c5be6aec157)

This is a default behavior in `1.7b`

**Reply by Axenide** [reply]
I know! And I'm really happy with the result. ;)

**Comment by DBLClick** [comment]
The one thing is side tabs behavior. In Arc Browser, Pinned Bookmarks are not loaded until you actually click to use them.  This to me is a good thing. But when I tab though my active tabs in the UI, all the unloaded tabs in the page are there and get loaded.  I would love the ability to have the pinned bookmarks to only be active, not in the tab UI, until they are actually loaded and in use.  This is lacking in Arc cloning.
